### PR TITLE
Update recaptcha-two.md

### PR DESCRIPTION
**Pull request by flcowboy7**
Adding new script tag to top of file to allow recaptcha to render.

# [DEVDOCS-](https://jira.bigcommerce.com/browse/DEVDOCS-)

## What changed?
* thing_that_changed

### CODE REVIEW

**Comment by uwikaiddi** [comment]
@flcowboy7, is there a ticket associated with this change?

**Comment by flcowboy7** [comment]
> 
> 
> @flcowboy7, is there a ticket associated with this change?

@uwikaiddi Case #: 04409216. They suggested I edit the page myself by coming here. Admittedly, I've never created a pull request before, so I'm not sure if I did it correctly. Basically, when using a legacy theme on BigCommerce with reCaptcha enabled, it is required for the designer to manually call the reCaptcha API script on the page.

**Comment by uwikaiddi** [comment]
@flcowboy7, gotcha. Wanted to make sure since we don't receive PRs for Legacy docs very much. I'll have a look. Thanks for submitting it!

**Comment by uwikaiddi** [comment]
@flcowboy7, what's your real name so I can throw some Bonusly your way?

**Comment by flcowboy7** [comment]
> 
> 
> @flcowboy7, what's your real name so I can throw some Bonusly your way?

@uwikaiddi Brandon (what's Bonusly?)

File: docs/legacy/blueprint-themes/recaptcha-two.md [hunk 1]
```diff
@@ -86,8 +86,27 @@ var RecaptchaOptions = {
 ```
 
 ### reCAPTCHA Script to Update
+(Reference: https://developers.google.com/recaptcha/docs/display)
```
Comment (uwikaiddi) [inline docs/legacy/blueprint-themes/recaptcha-two.md:89]:
```suggestion

```

File: docs/legacy/blueprint-themes/recaptcha-two.md [hunk 2]
```diff
@@ -86,8 +86,27 @@ var RecaptchaOptions = {
 ```
 
 ### reCAPTCHA Script to Update
+(Reference: https://developers.google.com/recaptcha/docs/display)
 
-Next, in each of the three template files, replace the "before" code block just below with the simplified "after" code that follows.
+Next, in each of the three template files, add the following script tage to the top of the page:
```
Comment (uwikaiddi) [inline docs/legacy/blueprint-themes/recaptcha-two.md:91]:
```suggestion
In each of the three template files, add the following script tag to the top of the page.

```

**Pull request merged**